src/UseCases/ValidateMembershipFee/ValidateFeeRequest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\MembershipContext\UseCases\ValidateMembershipFee;
+
+/**
+ * @license GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
```

Comment:
The outcome of the "@author tag yes or no" discussion was that in our team we don't use them.

Reply:
Ugh sure. I did not consciously put it here. I just copied another file because I'm too lazy to type the first few lines of the file.